src/main/java/healthy/lifestyle/backend/mentals/controller/MentalController.java

```diff
@@ -41,4 +45,55 @@ public ResponseEntity<MentalResponseDto> getCustomMentalById(@PathVariable("ment
+            @RequestParam(required = false) @DescriptionValidation String description,
+            @RequestParam(required = false) Long mentalTypeId,
+            @RequestParam(required = false, defaultValue = "title") String sortField,
+            @RequestParam(required = false, defaultValue = "ASC") String sortDirection,
+            @RequestParam(required = false, defaultValue = "10") int pageSize,
+            @RequestParam(required = false, defaultValue = "0") int pageNumber) {
+        Long userId = null;
+        if (isCustom == null || isCustom)
+            userId = authService.getUserIdFromAuthentication(
+                    SecurityContextHolder.getContext().getAuthentication());
+        Page<MentalResponseDto> dtoPage = mentalService.getMentalWithFilter(
+                isCustom, userId, title, description, mentalTypeId, sortField, sortDirection, pageNumber, pageSize);
+        return ResponseEntity.ok(dtoPage);
+    }
+
+    @GetMapping("/default")
+    public ResponseEntity<Page<MentalResponseDto>> getDefaultMentals(
+            @RequestParam(required = false) @TitleValidation String title,
+            @RequestParam(required = false) @DescriptionValidation String description,
+            @RequestParam(required = false) Long mentalTypeId,
+            @RequestParam(required = false, defaultValue = "title") String sortField,
+            @RequestParam(required = false, defaultValue = "ASC") String sortDirection,
+            @RequestParam(required = false, defaultValue = "10") int pageSize,
+            @RequestParam(required = false, defaultValue = "0") int pageNumber) {
+        Page<MentalResponseDto> dtoPage = mentalService.getMentalWithFilter(
+                false, null, title, description, mentalTypeId, sortField, sortDirection, pageNumber, pageSize);
+        return ResponseEntity.ok(dtoPage);
+    }
+
+    @GetMapping("/custom")
+    @PreAuthorize("hasRole('ROLE_USER')")
+    public ResponseEntity<Page<MentalResponseDto>> getCustomMentals(
+            @RequestParam(required = false) @TitleValidation String title,
+            @RequestParam(required = false) @DescriptionValidation String description,
+            @RequestParam(required = false) Long mentalTypeId,
+            @RequestParam(required = false, defaultValue = "title") String sortField,
+            @RequestParam(required = false, defaultValue = "ASC") String sortDirection,
+            @RequestParam(required = false, defaultValue = "10") int pageSize,
+            @RequestParam(required = false, defaultValue = "0") int pageNumber) {
+        Long userId = authService.getUserIdFromAuthentication(
+                SecurityContextHolder.getContext().getAuthentication());
+        Page<MentalResponseDto> responseDtoPage = mentalService.getMentalWithFilter(
+                true, userId, title, description, mentalTypeId, sortField, sortDirection, pageNumber, pageSize);
+        return ResponseEntity.ok(responseDtoPage);
+    }
```

Comment:
Seems okay, but I don't know why do you need `getCustomMentals()` if you have pre-authorized `getMentalsWithFilter()`.